qcodes/instrument_drivers/Lakeshore/Model_325.py

```diff
@@ -471,7 +444,7 @@ def __init__(self, parent: 'Model_325', name: str, loop: int) -> None:
         self.add_parameter(
             "resistance",
             get_cmd=f"HTRRES? {self._loop}",
-            set_cmd=f"HTRRES {self._loop} {{}}",
+            set_cmd=f"HTRRES {self._loop}, {{}}",
```

Comment:
oh) this should've been a separate PR but it's ok, just please mention this change in the description (and perhaps in the title)

Reply:
I have updated the description and title.